data/Language/Java.yaml

```diff
@@ -0,0 +1,15 @@
+identifier: Java
+# wikidata should be Qid only
+wikidata: Q251
+full_name: Java
+aliases:
```

Comment:
Aliases isn't needed if its same as full name. maybe you can change it to `java`

Reply:
Done.